rfcs/react-components/convergence/custom-styling.md

```diff
@@ -0,0 +1,214 @@
+# RFC: Custom Styling
+
+Geoff Cox
+Ben Howell
+Mason Tejera
+Brandon Thomas
+
+## Summary
+
+Fluent UI React v9 is built today to enforce adopting the Fluent 2 design styles. Partners creating component
+libraries don't know how provide their customers the ability to deviate from Fluent 2 design like they could
+with v0, v7, or v8.
+
+We need to define a custom styling escape hatch mechanism or pattern for these partners.
+
+## Background
+
+Fluent UI React v9 strongly conforms to the Fluent 2 Design to provide consistency in component behavior, layout,
+and style. Most partners are expected to adopt the Fluent 2 Design and remove custom styling that deviates from the
+design direction for Office.
+
+Some of our partner teams are building component libraries on top of Fluent UI React.
+They have customers who need to diverge from the Fluent 2 design for both the partners components and our components.
```

Comment:
I think just about every team I've worked with has to diverge from Fluent in some form.